compose.yaml

```diff
@@ -0,0 +1,31 @@
+services:
+  odoo:
+    build: .docker/odoo
+    depends_on:
+      - db
+    environment:
+      - SSH_AUTH_SOCK=/ssh-agent
+      - LANG=en_US.UTF-8
+      - LC_ALL=en_US.UTF-8
+    ports:
+      - "8069"
+      - "8072"
+    privileged: true
+    volumes:
+      - $SSH_AUTH_SOCK:/ssh-agent
```

Comment:
or what value should be set in `SSH_AUTH_SOCK` env var?

Reply:
`SSH_AUTH_SOCK` should be already defined automatically in the host, if it is running an ssh agent. `/ssh-agent` is just the path for the host's ssh socket inside the container.

I was just forwarding the host's ssh socket into the container, so that SSH inside the container worked as it would in the host. I removed it since I am no longer sure if this will be necessary, but we can review it